app/frontend/wailsjs/go/main/App.d.ts

```diff
@@ -1,6 +1,6 @@
 // Cynhyrchwyd y ffeil hon yn awtomatig. PEIDIWCH Â MODIWL
```

Comment:
The `wailsjs` contents are automatically generated, so shouldn't be automatically formatted. It says `DO NOT EDIT`. This folder should be excluded for prettier.